package.json

```diff
@@ -78,7 +78,7 @@
     "react-transition-group": "^4.4.5",
     "react-virtualized-auto-sizer": "1.0.5",
     "recoil": "0.4.1",
-    "sharp": "^0.30.5",
+    "sharp": "^0.31.2",
```

Comment:
had to be updated due to issue with ARM machines https://github.com/lovell/sharp/issues/2340